README.md

```diff
@@ -0,0 +1,45 @@
+# terraform-aws-cross-account-role
+A Terraform module to create an IAM role for cross-account use. This module creates the role in the destination account, but does not configure access in the source account.
+
+# Usage
+```hcl
+# Account 111111111111 configuration
+
+# Creates arn:aws:iam::111111111111:role/CrossAccountDeveloper
+module "cross_account_role" {
+  source                      = "github.com/azavea/terraform-aws-cross-account-role?ref=0.1.0"
+  name                        = "CrossAccountDeveloper"
+  principal_arns              = ["222222222222","arn:aws:iam::333333333333:user/MyUser"]
```

Comment:
Is `222222222222` equivalent to `arn:aws:iam::222222222222:root`, or is there a difference? I've only ever used the latter to reference an entire account as a principal.

Reply:
They're equivalent. From what I saw in Terraform diffs, `222222222222` actually gets expanded to the `arn:aws:iam::222222222222:root` when the IAM policy is created.